content/en/news/support/advance-notice-opencensus-deprecation/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Advance notice of Istio deprecating OpenCensus
+subtitle: Support Announcement
+description: Advance notice of Istio deprecating OpenCensus.
+publishdate: 2023-08-17
+---
+
+OpenCensus was [sunset](https://opentelemetry.io/blog/2023/sunsetting-opencensus) on July 31st, 2023.
+
+The sunset of OpenCensus will impact Istio's usage of OpenCensus in tracing, logging, and metrics. Istio
+plans to use OpenTelemetry exporters with the OpenTelemetry collector to provide matching telemetry capabilities.
```

Comment:
Please remove this. There is no consensus in Istio that OTEL collectors will be used in Istio.